scm/driver/bitbucket/git.go

```diff
@@ -44,7 +45,12 @@ func (s *gitService) FindCommit(ctx context.Context, repo, ref string) (*scm.Com
 			ref = branch.Sha // replace ref with sha
 		}
 	}
-	path := fmt.Sprintf("2.0/repositories/%s/commit/%s", repo, ref)
+	var path string
+	if strings.Contains(ref, "/") {
```

Comment:
in some cases I have seen the ref resolving to `refs/heads/branchname`, that's not gonna be a problem?

Reply:
No, that will get resolved by separate function FindBranch()